web-ts/src/routes/TuNewsDetailPage.tsx

```diff
@@ -18,6 +18,7 @@ import { FailureSwitcher } from '../components/FailureSwitcher'
 import Page from '../components/Page'
 import styled from 'styled-components'
 import TunewsIcon from '../assets/TunewsActiveLogo.png'
+import { parseHTML } from 'api-client/src/utils/helpers'
```

Comment:
You can also just import and export `parseHTML` in the `api-client/src/index` so that you can import it here from just `api-client` instead of the full path.